cmd/commands/install.go

```diff
@@ -47,32 +49,53 @@ func installCmd(ctx *cli.Context) error {
 		return installAll()
 	}
 
+	args := ctx.Args()
 	sdkArg := ctx.Args().First()
 	if sdkArg == "" {
 		return cli.Exit("sdk name is required", 1)
 	}
-	argArr := strings.Split(sdkArg, "@")
-	argsLen := len(argArr)
+
 	manager := internal.NewSdkManager()
 	defer manager.Close()
-	if argsLen > 2 {
-		return cli.Exit("sdk version is invalid", 1)
-	} else {
-		var name string
-		var version internal.Version
-		if argsLen == 2 {
-			name = strings.ToLower(argArr[0])
-			version = internal.Version(argArr[1])
+
+	for i := 0; i < args.Len(); i++ {
```

Comment:
The purpose of a for loop is?

Reply:
we can now install multiple version by one command:
```
vfox install nodejs@20 nodejs@21
```

Reply:
You shouldn't return in a for loop.

Reply:
submit a new implementation

Reply:
> we can now install multiple versions by one command:

This usage needs to be marked in the documentation

Reply:
done